blog-posts/return-to-office-interview.md

```diff
@@ -0,0 +1,142 @@
+Damit einher geht die bessere Vereinbarkeit von Familie und Beruf. Ich kann mich noch an Projekte erinnern, wo es Pflicht war spätestens um 9:00 Uhr vor Ort zu sein und in die Tasten zu hauen.
+In der Zeit, musste ich davor aber auch jeden Morgen meinen Sohn zur Kita bringen. Das hieß dann morgens mit S-Bahn, Bus und Fussweg von Milbertshofen im Norden nach Untermenzingen in die Kita. Danach dann mit Bus und S-Bahn zur Arbeit. Das war schon ein brutaler Stress, den ich jeden Morgen hatte. Praktisch hieß das, dass ich spätestens um 7:00 Uhr das Haus verlassen musste, damit ich auch garantiert um 9:00 Uhr pünktlich im Büro sein konnte.
+Wenn so ein Zwang wegfällt, ist man ganz automatisch auch entspannter und kann mehr Energie in die Arbeit investieren.
+
+**Was plant ihr für die “Rückkehr zum Arbeitsplatz”? Wird es Vorgaben geben, was den Arbeitsort angeht?**
+
+**Gholam:** Das werden wir unseren Mitarbeitern komplett frei stellen. Wir wollen da keine festen Vorgaben machen. Von Vollzeit Remote, Vollzeit Büro und allem was dazwischen denkbar ist, werden wir den Mitarbeitern die Freiheit geben, ihren Arbeitsrhythmus so zu planen, dass sie gut arbeiten können.
+
+**Eric:** Als einzige Ausnahme sehen wir hier Senior- bzw Manager-Aufgaben. Ein Beispiel: Zum Mentoring gehört ja nicht nur das Weitergeben von Hard Skills, sondern auch von Soft Skills und das funktioniert in persona einfach erfahrungsgemäß besser als remote.
+Aber ich glaube, das pendelt sich ohne Vorgaben unsererseits schon richtig ein, denn es ist immer wahnsinnig schön, den Lerneffekt und die Freude bei einem Apprentice (so nennen wir unsere Junioren) zu erleben und das geht ja nur gemeinsam. Das Verhältnis zu unseren Mitarbeitern ist geprägt von Vertrauen und der Übergabe von Verantwortung und damit sind wir bisher immer sehr gut gefahren. Unser Vertrauen muss man sich nicht verdienen, man kann es nur verlieren.
+
+**Plant ihr auch Stellen Vollzeit Remote zu besetzen?**
+
+**Gholam:** Ja, das ist auf jeden Fall etwas was wir in Betracht ziehen. Du bist da bereits der erste Mitarbeiter, bei dem bei der Einstellung schon feststand, dass du Vollzeit Remote von Fulda aus arbeiten wirst. Und weil das so gut geklappt hat, ziehen wir bei der Besetzung von offenen Stellen auch jetzt schon Leute in Betracht, die nicht in München leben und auch nicht für die Arbeit dorthin umziehen wollen.
+Im Detail ist das aber noch etwas, was wir für uns erarbeiten müssen… Welche Voraussetzungen zum Beispiel geschaffen werden müssen, um einen Kollegen aus Brasilien im Team begrüßen zu dürfen, das wissen wir noch nicht. Deswegen suchen wir im ersten Schritt auch erstmal Mitarbeiter im deutschen Raum.
+
```

Comment:
Please do not stop without any closing words. You absolutely need to say good by to the reader. It feels super uncomfortable as a reader that this ends like that. A simple "We have reached the end of the interview. Thank you both." or such can help a ton here already. I'm also sure that you can do even do more here while being at it ? Don't you think? Did you omit by purpose ? Thanks for any inisghts